plugins/modules/foreman_templates_import.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Anton Nesterov (@nesanton)
+#
+# This file is part of Ansible
```

Comment:
no it's not, can you please update to the header we (now) use in all the other plugins? :)